core/src/main/java/com/fincity/saas/core/functions/storage/ReadPageStorageObject.java

```diff
@@ -93,6 +99,9 @@ public FunctionSignature getSignature() {
 						COUNT, Parameter.of(COUNT, Schema.ofBoolean(COUNT)
 								.setDefaultValue(new JsonPrimitive(true))),
 
+					    SORT, Parameter.of(SORT,Schema.ofArray(SORT)
```

Comment:
You need to provide the shape of the objects in the array, It cannot be simply array of anything. The shape of the object like the Order object specified here https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Sort.Order.html.